python/taichi/lang/common_ops.py

```diff
@@ -0,0 +1,56 @@
+class TaichiOperations:
+    def __neg__(self):
+        import taichi as ti
+        return ti.neg(self)
+
+    def __abs__(self):
+        import taichi as ti
+        return ti.abs(self)
+
+    def __add__(self, other):
+        import taichi as ti
+        return ti.add(self, other)
+
+    __radd__ = __add__
```

Comment:
```suggestion
    def __radd__(self, other):
        import taichi as ti
        return ti.add(other, self)
```

Reply:
This suggestion makes more sense if we run into `+` operations that are not commutative in the future.